### ISSUE TITLE
[Bug]: Some trackers only show 1 torrent

### ISSUE THREAD

**Issue by terrytw** [VueTorrent/VueTorrent#1929]
### Bug description

There are some trackers that have different tracker URL for different torrents.

e.g.
https://daydream.dmhy.best/announce?secure=adgsdsagd12412441asgd39asdgasd
https://daydream.dmhy.best/announce?secure=124142sadgdsgd325423sadgsdagdfs

They have the same domain name, but different query key value. 

In vue, when filtering by tracker, only 1 entry shows up. 

This has been fixed in original WebUI, see https://github.com/qbittorrent/qBittorrent/pull/18190 and https://github.com/qbittorrent/qBittorrent/pull/19062

### Expectations

All torrent entries show up when filtering using tracker

### Relevant debug informations

N/A

### Steps to reproduce

If you are on u2.dmhy.org, or teamos, you can download multiple torrents and filter by tracker to see it. 

### How did you install VueTorrent?

Downloaded from GitHub releases

### Environment description

```plain text
Device type: (NAS)
OS: (Linux)
Browser (name and version): chrome 118

PWA: No
Running inside a container: Yes
```

### qBittorrent version

4.6.5

### VueTorrent version

backend 2.1.1

### Agreements

- [X] The bug still occurs on the nightly build
- [X] The bug still occurs on the latest version of qBittorrent
- [X] A similar issue doesn't already exist
- [X] I checked the DevTools and included any errors that may show up

**Comment by Larsluph** [comment]
I'm not sure to understand what's the problem here...

Do you want to see the entire path or just the hostname?

Because we're already filtering the hostname based on a previous issue: #1833

**Comment by terrytw** [comment]
![1](https://github.com/user-attachments/assets/ea678399-80bb-40c3-9832-2cfbf23e1a2e)

![2](https://github.com/user-attachments/assets/96381652-a701-432b-81b3-113f9c38952a)

You can see from the screenshots that when filtering using tracker in vuetorrent, I only got 1 entry, but in original webUI I got 300 entries.

**Comment by Larsluph** [comment]
Then there may be another issue that prevents other torrents from showing up.
Please follow [Support page](https://github.com/VueTorrent/VueTorrent/wiki/Support) to ensure no errors are there.
For better reproductibility, I would need an extract of at least 2 torrents that should show under the same tracker and that aren't. All torrent data are inside the `maindata request`, support page will also show you how to proceed.

**Comment by terrytw** [comment]
1. All network request but one is successful. There is one that is blocked by adblocker I guess
`https://ipinfo.io/*.*.*.*/json`

2 I have sent the information you requested to your Email. Since there is private tracker information and their rule usually asks for users to not disclose tracker announce URL, I am sending it privately through Email.

Please let me know if you need any other information. Thanks!

**Comment by Larsluph** [comment]
I can successfully reproduce the bug using your extract. I'll see what I can do about it.

**Comment by terrytw** [comment]
Thank you for the swift fix, much appreciated!

I presume next release of the backend will include all the latest changes in this repository right? I skimmed the github action of backend repo, it seems that every new release will pull the latest code from here and build from scratch.

**Comment by Larsluph** [comment]
VueTorrent and the backend have independent release cycles. With the latest version of the backend, a cron job runs periodically to update VueTorrent automatically, you can configure the interval as you want.

Download and setup steps are launched at runtime only, so you'll always have the latest version of VueTorrent.
